Patch by Robert Scheck <robert@fedoraproject.org> for iftop <= 1.0pre4 to avoid a
regression that would reopen https://bugzilla.redhat.com/show_bug.cgi?id=546032.

--- iftop-1.0pre4/configure.ac			2014-01-19 21:28:31.000000000 +0100
+++ iftop-1.0pre4/configure.ac.ncursesw		2014-01-20 00:12:54.000000000 +0100
@@ -353,7 +353,7 @@
 for curseslib in ncursesw curses ncurses ; do
     LIBS="$oldLIBS -l$curseslib"
     AC_TRY_LINK([
-#include <$curseslib.h>
+#include <curses.h>
         ], [
         mvchgat(0, 0, 1, A_REVERSE, 0, NULL)
         ], [
--- iftop-1.0pre4/configure			2014-01-19 21:32:58.000000000 +0100
+++ iftop-1.0pre4/configure.ncursesw		2014-01-20 00:13:15.000000000 +0100
@@ -5745,7 +5745,7 @@
     cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
-#include <$curseslib.h>
+#include <curses.h>
 
 int
 main ()
